packages/esm-patient-immunizations-app/src/immunizations/immunizations-form.workspace.tsx

```diff
@@ -314,6 +319,25 @@ const ImmunizationsForm: React.FC<DefaultPatientWorkspaceProps> = ({
               </ResponsiveWrapper>
             </section>
           )}
+          <section>
+            <ResponsiveWrapper>
+              <Controller
+                name="note"
+                control={control}
+                render={({ field: { onChange, value } }) => (
+                  <div className={styles.row}>
+                    <TextInput
```

Comment:
I'd expect this to be a [Textarea](https://react.carbondesignsystem.com/?path=/docs/components-textarea--overview). I'd also expect it to have the `enableCounter` and `maxCount` props specified.

Reply:
Could you update the screenshot to reflect this?